src/components/common/Button/index.jsx

```diff
@@ -0,0 +1,34 @@
+import React from 'react' ;
+import PropTypes from 'prop-types' ;
+import styles from './Button.module.scss';
+
+const Button = ({onClick,children,...props})=>{
+  return <button className={styles.btn} onClick={onClick} {...props}>{children}</button>
+}
+Button.propTypes = {
+  onClick : PropTypes.func.isRequired,
+  children:PropTypes.node
+
+}
+
+export const CompleteButton = ({onClick,children,...props})=>{
+  return <button className={styles.completeBtn} onClick={onClick} {...props}>{children}</button>
+}
+CompleteButton.propTypes = {
+  onClick : PropTypes.func.isRequired,
+  children:PropTypes.node
+}
+export const DeleteButton = ({onClick,children,...props})=>{
```

Comment:
Having the same component copied several time just to change a class name is a very bad practice. Use a prop instead